src/commands/update.test.ts

```diff
@@ -0,0 +1,68 @@
+import type { User } from 'discord.js';
+
+// Overwrite the exec function
+const mockExec = jest.fn((command: string, callback: () => void) => {
+	callback();
+});
+const cp = jest.requireActual<typeof import('node:child_process')>('node:child_process');
+jest.mock('node:child_process', () => ({
+	...cp,
+	exec: mockExec,
+}));
+
+import { update } from './update';
+
+describe('update', () => {
+	const ADMINISTRATORS_VARIABLE = 'ADMINISTRATORS';
+	const adminUser: User = {
+		username: 'TheHost',
+		id: '1',
+	} as unknown as User;
+	const otherUser: User = {
+		username: 'TheCitizen',
+		id: '3',
+	} as unknown as User;
+	const mockReplyPrivately = jest.fn<Promise<void>, [content: string]>();
+	const mockEditReply = jest.fn<Promise<void>, [content: string]>();
+
+	let context: TextInputCommandContext;
+
+	beforeEach(() => {
+		// Overwrite the environment variable for token
+		const mockAdmins = '1,2';
+		process.env[ADMINISTRATORS_VARIABLE] = mockAdmins;
```

Comment:
It is customary, when overwriting globals, to store the original global and do the overwrite in `beforeEach`, then restore the global in `afterEach`. This way, tests don't need to make assumptions about shared state with respect to other tests.

Reply:
done